### PR TITLE
Merging "main.js" in the node backend into develop branch

### PR DESCRIPTION
**Pull request by jkuan1**
Ryan:
- created file and content inside

### CODE REVIEW

**Comment by jkuan1** [comment]
@ThienH7 Please ask Ryan any questions/comments on the structure of the code before I merge

File: back-end/sample.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ visit localhost:5000 to see the message!
 const http = require('http');
 
 const hostname = '127.0.0.1';
-const port = 5000;
+const port = 5069;
```
Comment (jkuan1) [inline back-end/sample.js]: @RyanmLin Please change back to 5000

File: back-end/main.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+})
+
+app.get('/example', (req, res) => {
+    //const firstName = req.query.firstName;
+    res.json({
+        UserName: 'Ryan Lin',
+        Password: 'Password',
+        Birthday: 'January 14, 1999',
+        Email: 'lin.ryan99@hotmail.com',
+        PhoneNumber: '604-446-6883'});
```
Comment (jkuan1) [inline back-end/main.js]: @RyanmLin  This is an open source repository why are you using actual information

File: back-end/user.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/*
+This is the user setting page. I want to return a json object here with
+the user's data
+*/
+
+//The constant variables we'll need
+const express = require('express')
+const app = express()
```
Comment (jkuan1) [inline back-end/user.js:10]: @RyanmLin is this necessary for this file? The current purpose seems to be the test2 function which do not use the express framework so why is it called here?

**Review by RyanmLin — APPROVED** [review]

**Pull request merged**